src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  font-family: $font-family;
+  padding: 16px;
+  transition: transform 1s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 0 0;
+  }
+
+  &__header {
+    color: $color-black;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 40px 0 0;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the margin should be set using shorthand properties. Instead of specifying `margin-top` separately, use `margin: 40px 0 0;`.